[2025-02-24T05:05:18.419+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T05:05:18.446+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_1.second_task scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-02-24T05:05:18.461+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_1.second_task scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-02-24T05:05:18.462+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T05:05:18.504+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-02-01 00:00:00+00:00
[2025-02-24T05:05:18.513+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1867) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T05:05:18.513+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_1', 'second_task', 'scheduled__2025-02-01T00:00:00+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpqflvlot5']
[2025-02-24T05:05:18.515+0000] {standard_task_runner.py:72} INFO - Started process 1877 to run task
[2025-02-24T05:05:18.515+0000] {standard_task_runner.py:105} INFO - Job 129: Subtask second_task
[2025-02-24T05:05:18.576+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag_1.second_task scheduled__2025-02-01T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T05:05:18.666+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag_1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-01T00:00:00+00:00'
[2025-02-24T05:05:18.667+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T05:05:18.682+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T05:05:18.683+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is task 2, and will be run after task 1.']
[2025-02-24T05:05:18.691+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T05:05:18.693+0000] {subprocess.py:106} INFO - hey, this is task 2, and will be run after task 1.
[2025-02-24T05:05:18.694+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T05:05:18.723+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T05:05:18.724+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag_1, task_id=second_task, run_id=scheduled__2025-02-01T00:00:00+00:00, execution_date=20250201T000000, start_date=20250224T050518, end_date=20250224T050518
[2025-02-24T05:05:18.771+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T05:05:18.785+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
